solr/server/solr/configsets/sample_techproducts_configs/conf/solrconfig.xml

```diff
@@ -1597,8 +1601,18 @@
      <valueSourceParser name="myfunc" 
                         class="com.mycompany.MyValueSourceParser" />
     -->
-    
-  
+  <!-- Learning to rank query parser
```

Comment:
How about something like

       You'll need to set the solr.ltr.enabled system property
       when running solr to run with ltr enabled:
       -Dsolr.ltr.enabled=true

here i.e. similar to the clustering component's config.